### ISSUE TITLE
Feature Request: add timezone argument

### ISSUE THREAD

**Issue by HJAllen** [gaborcsardi/parsedate#25]
Is it possible to add a timezone argument or option so that returned objects can have a timezone other than the system timezone? So, if I parse dates in UTC they return as UTC not system tz. I see you have the helper function `with_tz` but this changes the absolute time. `lubridate::force_tz` has the functionality but as you mention elsewhere, it requires loading a lot of other packages.

**Comment by sckott** [comment]
+1 on this @gaborcsardi thinking of using this in a pkg, but for my use case I just want users to be able to get date times back in the same date time given, e.g., 

```r
Sys.getenv("TZ")
#> "US/Pacific"
parsedate::parse_date("January 4, 1981")
#> [1] "1981-01-03 16:00:00 PST"
```

Where the parsed date goes back my TZ. I'd like to be able to set TZ = utc, so the user supplied date time stays in that TZ

**Comment by gaborcsardi** [comment]
Can you just call `as.POSIXlt()` at the end?

**Comment by sckott** [comment]
thanks, that seems to work nicely

**Comment by gaborcsardi** [comment]
Yeah, I think I'll just add an example that converts the result to the local time zone.
